src/ssvc/decision_points/safety_impact.py

```diff
@@ -78,8 +144,24 @@
 )
 
 
+SAFETY_IMPACT_2 = SsvcDecisionPoint(
+    name="Safety Impact",
+    description="The safety impact of the vulnerability. (based on IEC 61508)",
+    key="SI",
+    version="2.0.0",
+    values=(
+        NEGLIGIBLE,
+        MARGINAL,
+        CRITICAL,
+        CATASTROPHIC_2,
+    ),
+)
+
+
 def main():
-    print(SAFETY_IMPACT_1.to_json(indent=2))
+    versions = (SAFETY_IMPACT_1, SAFETY_IMPACT_2)
+
+    print_versions_and_diffs(versions)
```

Comment:
Running this through print_versions_and_diffs confirmed it deserved a new major version